js/core/utils/date.js

```diff
@@ -672,6 +681,7 @@ const dateUtils = {
     getLastMonthDate: getLastMonthDate,
     getFirstMonthDate: getFirstMonthDate,
     getFirstWeekDate: getFirstWeekDate,
+    getWeekNumber: getWeekNumber,
```

Comment:
Can we use a short syntax like this?

`const dateUtils = {
     getWeekNumber,
}`

Reply:
nope, untill we export one object. I guess we can refactor these utils to seperately export all function we need, but not sure it's we should do now, as long as other teams can be "tied" with current syntax